tests/phpunit/Civi/Extendedreport/BaseTestClass.php

```diff
@@ -356,7 +358,26 @@ public function getAllNonLoggingReports(): array {
    * @noinspection PhpUnhandledExceptionInspection
    */
   public function getAllReports(): array {
-    return (array) Managed::get(FALSE)->addWhere('module', '=', 'nz.co.fuzion.extendedreport')->execute();
+    if (!defined('ASSUME_GRANT_INSTALLED') || !ASSUME_GRANT_INSTALLED) {
```

Comment:
@eileenmcnaughton @totten I had to do things like this because this is used as part of data providers which is evaluated before CiviCRM gets booted so we cannot call any APIs either v3 or v4 before CiviCRM is booted